### PR TITLE
71 update model card

### LINKED ISSUE #70: Add Model Card to repo
- [Something similar to ](https://huggingface.co/docs/hub/model-cards)Model
 Cards.

### PR DESCRIPTION
**Pull request by ChristinaLast**
# Description

Adds additional changes mentioned by @jit-1 in #70 
Fixes #70 

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How has this been tested?

Please describe the tests that you ran to verify your changes.

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: .github/MODEL_CARD.md [hunk 1]
```diff
@@ -132,7 +149,7 @@ The model achieved F1-Scores of 0.66 and 0.71 for Gradient Boosting and Random F
 
 The model demonstrates robust performance in predicting PM2.5 concentrations across diverse regions and conditions.
 
-## Model Examination [optional]
+## Model Examination
```
Comment (jit-1) [inline .github/MODEL_CARD.md]: Is not required to have the head as no details are added.

File: .github/MODEL_CARD.md [hunk 2]
```diff
@@ -80,21 +93,22 @@ Use the code below to get started with the model.
...
+#### Preprocessing
 
 Data preprocessing involves cleaning and integrating data from multiple sources, extracting relevant features, and handling missing values.
 
 #### Training Hyperparameters
 
 - **Training regime:** fp32 precision
 
-#### Speeds, Sizes, Times [optional]
+#### Speeds, Sizes, Times
```
Comment (jit-1) [inline .github/MODEL_CARD.md]: Is not required to have the head as no details are added.

File: .github/MODEL_CARD.md [hunk 3]
```diff
@@ -162,13 +179,24 @@ The model can be deployed on any Linux service or cloud environment. Currently,
...
+- **SSD:** 100 GB
+- **vCPUs:** 4
+
+These specifications provide a balance of computational power, memory, and storage to efficiently handle the training and deployment of the model. The use of AWS EC2 instances ensures scalability and flexibility, allowing for adjustments based on workload requirements.
 #### Software
 
 The model uses Python libraries such as scikit-learn, pandas, and numpy for data processing and model training.
 
-## Citation [optional]
+## Citation
```
Comment (jit-1) [inline .github/MODEL_CARD.md:183]: Is not required to have the head as no details are added.

File: .github/MODEL_CARD.md [hunk 4]
```diff
@@ -44,7 +49,7 @@ This model leverages machine learning techniques to predict PM2.5 concentrations
 
 The model can be directly used to predict PM2.5 concentrations for specific locations and dates by using the provided CLI tool. It is designed for environmental researchers, policy makers, and public health officials to monitor and predict air quality.
 
-### Downstream Use [optional]
+### Downstream Use
```
Comment (jit-1) [inline .github/MODEL_CARD.md]: Is not required to have the head as no details are added.

File: .github/MODEL_CARD.md (hunk 3 above)
Comment (ChristinaLast) [reply .github/MODEL_CARD.md:183]: there are details there
```

## Citation

```bibtex
@INPROCEEDINGS{2022AGUFMIN42C0346L,
       author = {{Last}, Christina and {Pramanik}, Prithviraj},
        title = "{Using Open Data to Understand Air Pollution Exposure in Resource-Constrained Environments}",
    booktitle = {AGU Fall Meeting Abstracts},
         year = 2022,
       volume = {2022},
        month = dec,
          eid = {IN42C-0346},
        pages = {IN42C-0346},
       adsurl = {https://ui.adsabs.harvard.edu/abs/2022AGUFMIN42C0346L},
      adsnote = {Provided by the SAO/NASA Astrophysics Data System}
}
```
```bibtex
@article{pramanik2023remote,
  title={Remote Sensing and Machine Learning based models to understand Air Pollution Exposure in Resource-Constrained Environments},
  author={Pramanik, Prithviraj and Last, Christina},
  journal={AGU23},
  year={2023},
  publisher={AGU}
}
```

Comment (ChristinaLast) [reply .github/MODEL_CARD.md:183]: there are details there
```

## Citation

```bibtex
@INPROCEEDINGS{2022AGUFMIN42C0346L,
       author = {{Last}, Christina and {Pramanik}, Prithviraj},
        title = "{Using Open Data to Understand Air Pollution Exposure in Resource-Constrained Environments}",
    booktitle = {AGU Fall Meeting Abstracts},
         year = 2022,
       volume = {2022},
        month = dec,
          eid = {IN42C-0346},
        pages = {IN42C-0346},
       adsurl = {https://ui.adsabs.harvard.edu/abs/2022AGUFMIN42C0346L},
      adsnote = {Provided by the SAO/NASA Astrophysics Data System}
}
```
```bibtex
@article{pramanik2023remote,
  title={Remote Sensing and Machine Learning based models to understand Air Pollution Exposure in Resource-Constrained Environments},
  author={Pramanik, Prithviraj and Last, Christina},
  journal={AGU23},
  year={2023},
  publisher={AGU}
}
```

**Comment by ChristinaLast** [comment]
All these headings had a preliminary section underneath them, so I am unsure why you have suggested there is no details.

**Review by jit-1 — APPROVED** [review]

**Pull request merged**